server/database/dbTables.js

```diff
@@ -0,0 +1,95 @@
+import pool from "./dbindex";
+
+const databaseTables = () => {
+
+    const usersTable = `
+    CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(25) NOT NULL UNIQUE,
+    password VARCHAR(250) NOT NULL,
+    first_name  VARCHAR(25) NOT NULL,
+    last_name VARCHAR(25) NOT NULL,
+    id_group INT
```

Comment:
What is this particular column for?

Reply:
It was found unnecessary and then removed.